pkg/scheduler/testing/wrappers.go

```diff
@@ -477,6 +478,17 @@ func (n *NodeWrapper) Label(k, v string) *NodeWrapper {
 	return n
 }
 
+// Taint add a taint to node with the current time.
+func (n *NodeWrapper) Taint(k, v, e string) *NodeWrapper {
+	n.Spec.Taints = append(n.Spec.Taints, v1.Taint{
+		Key:       k,
+		Value:     v,
+		Effect:    v1.TaintEffect(e),
+		TimeAdded: &metav1.Time{Time: time.Now()},
```

Comment:
is this mandatory?

Reply:
no, I just add it as a default value.

Reply:
then let's remove it and add it when necessary.

Reply:
sounds good.